ci/python_310.yaml

```diff
@@ -6,6 +6,7 @@ channels:
 dependencies:
     - python=3.10
     - configargparse
+    - fiona
```

Comment:
`fiona` comes with geopandas always, so not necessarily needed here. However, I'm okay having it listed.

Reply:
That’s true. Let’s kick it from here, already `geopandas` itself is a bit of a stretch, since we do not (yet) import it anywhere (but it really IS needed for any sensible use of r5py)

Reply:
no, actually, thinking about it: we ARE now importing it in the tests, so I’d rather keep it included explicitely, so we don’t run into any issues with the test runners.